### DISCUSSION TITLE
Rokonet / Orbit-Pro alarm system

### DISCUSSION THREAD

**Discussion by natiz** [taligentx/dscKeybusInterface#118] [dscKeybusInterface Discussion]
Has anyone tested this on a Rokonet Orbit-Pro system (RP-296)? it seems to also use a 4-wire bus and is comprised of a main unit, and 'slave' modules - keypads, expansion modules, etc.

Installation manual: http://www.icas.no/no/filer/Orbit_Pro_Installation_Manual.pdf
User manual: http://www.riscogroup.com/uk/downloader/9157/39438b30c5865855a9651b894bbd3aee

**Comment by taligentx** [comment]
Hi @natiz - late response, the Rokonet doesn't look like a re-branded DSC panel and so this library likely will not work with it.
